src/components/calcite-tab/calcite-tab.e2e.ts

```diff
@@ -1,12 +1,65 @@
 import { newE2EPage } from "@stencil/core/testing";
-import { HYDRATED_ATTR } from "../../tests/commonTests";
+import { defaults, HYDRATED_ATTR } from "../../tests/commonTests";
 
 describe("calcite-tab", () => {
-  it("renders", async () => {
-    const page = await newE2EPage();
+  const tabHtml = "<calcite-tab>A tab</calcite-tab>";
 
-    await page.setContent("<calcite-tab></calcite-tab>");
+  it("renders", async () => {
+    const page = await newE2EPage({ html: tabHtml });
     const element = await page.find("calcite-tab");
+    const styles = await element.getComputedStyle();
     expect(element).toHaveAttribute(HYDRATED_ATTR);
+    expect(styles["display"]).toEqual("none");
+    expect(styles["visibility"]).toEqual("visible");
+  });
+
+  it("has defaults", async () =>
+    defaults("calcite-tab", [
+      { propertyName: "tab", defaultValue: undefined },
+      { propertyName: "active", defaultValue: false },
+      { propertyName: "scale", defaultValue: undefined }
+    ]));
+
+  it("has block display when active", async () => {
+    const page = await newE2EPage({ html: "<calcite-tab active></calcite-tab>" });
+    const element = await page.find("calcite-tab");
+    const styles = await element.getComputedStyle();
+    expect(styles["display"]).toEqual("block");
+  });
+
+  describe("when nested within calcite-tabs component", () => {
+    it("should render with medium scale", async () => {
```

Comment:
I'm not entirely sure checking computed styles is necessary. IMO that's sort of what the visual regression testing is for.

Reply:
@paulcpederson wouldn't we would a need a story for the tabs in each scale for that to work?

Reply:
We could in theory accomplish this kind of testing with Screener Runner on demo pages, which would allow us to just render all the scales on a page that Screener can photograph and keep a record of.

Reply:
That would be awesome @eriklharper, you're referring to #2177 right?

Reply:
For sure, yep!